pkg/ctx/agentctx.go

```diff
@@ -25,16 +25,22 @@ import (
 	"google.golang.org/grpc/metadata"
 )
 
+// UserAgentHeader is the header used for the user agent
+const UserAgentHeader = "x-user-agent"
```

Comment:
Don't use a separate header. It's logged in other places so we should display the one we're adding